src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -140,7 +140,7 @@ public string Description
         [JsonIgnore]
         public string DisplayTypeName
         {
-            get { return Type.ToShortString(); }
+            get { return Type.ToString(); }
```

Comment:
does the protocore type has something else we can use like fullName or fullString - I'd rather not rely on the implementation of the toString on this type we control.

Reply:
<img width="671" alt="screen shot 2018-03-26 at 11 41 46 am" src="https://user-images.githubusercontent.com/8495197/37916785-bcfc8796-30ea-11e8-8ceb-a5a855c8c1d3.png">

Reply:
It has `Name`.

Reply:
`Name` will always represent the whole string I believe

Reply:
Yes. The test passed with `Name`. I will change it to `Name`